src/components/dataset/query/sidebar/panels/FilterPanel.jsx

```diff
@@ -230,7 +230,7 @@ export class FilterPanel extends React.PureComponent {
             ))}
         </div>
         <div className={clsx(classes.rowTwo, classes.snapshotBtnCntnr)}>
-          <Tooltip title={canLink ? '' : billingErrorMessage}>
+          <TerraTooltip title={canLink ? '' : billingErrorMessage}>
```

Comment:
Why not conditionally render the `TerraTooltip` based on `canLink`?

Reply:
I think it's a little more complicated than that b/c the tooltip component wraps other components and we always want to render the child components. Conditionally rendering TerraTooltip as the parent component would hide the child component. 
This is also out of scope of this ticket - I'm just making sure the formatting for the tooltip remains consistent across the app.